core/templates/pages/release-coordinator-page/features-tab/features-tab.component.ts

```diff
@@ -126,52 +259,57 @@ export class FeaturesTabComponent implements OnInit {
     return parsedDate.toLocaleDateString('en-US', options);
   }
 
-  getFeatureStageString(feature: FeatureFlag): string {
-    if (feature.featureStage === 'dev') {
+  getFeatureStageString(featureFlagVM: FeatureFlagViewModel): string {
+    if (featureFlagVM.featureStage === 'dev') {
       return 'Dev (can only be enabled on dev server).';
-    } else if (feature.featureStage === 'test') {
+    } else if (featureFlagVM.featureStage === 'test') {
       return 'Test (can only be enabled on dev and test server).';
     } else {
       return 'Prod (can only be enabled on dev, test and prod server).';
     }
   }
 
-  getFeatureValidOnCurrentServer(feature: FeatureFlag): boolean {
+  getFeatureValidOnCurrentServer(featureFlagVM: FeatureFlagViewModel): boolean {
     if (this.serverStage === this.DEV_SERVER_STAGE) {
       return true;
     } else if (this.serverStage === this.TEST_SERVER_STAGE) {
-      return feature.featureStage === this.TEST_SERVER_STAGE ||
-        feature.featureStage === this.PROD_SERVER_STAGE
+      return featureFlagVM.featureStage === this.TEST_SERVER_STAGE ||
+        featureFlagVM.featureStage === this.PROD_SERVER_STAGE
         ? true
         : false;
     } else if (this.serverStage === this.PROD_SERVER_STAGE) {
-      return feature.featureStage === this.PROD_SERVER_STAGE ? true : false;
+      return featureFlagVM.featureStage === this.PROD_SERVER_STAGE
+        ? true
+        : false;
     }
     return false;
   }
 
-  async updateFeatureFlag(feature: FeatureFlag): Promise<void> {
+  async updateFeatureFlag(featureFlagVM: FeatureFlagViewModel): Promise<void> {
```

Comment:
It is a bit confusing here (and possibly elsewhere) that updateX isn't updating a variable of type X (since, on the surface,  "feature flag" isn't the same as "feature flag VM").

Reply:
Right, I have updated it.